jsapp/js/components/common/koboRange.tsx

```diff
@@ -0,0 +1,124 @@
+import React from 'react'
+import bem, {makeBem} from 'js/bem'
+import './koboRange.scss'
+
+bem.KoboRange = makeBem(null, 'kobo-range')
+
+bem.KoboRange__values = makeBem(bem.KoboRange, 'values', 'div')
+bem.KoboRange__maxValue = makeBem(bem.KoboRange, 'max-value', 'span')
+bem.KoboRange__currentValue = makeBem(bem.KoboRange, 'current-value', 'span')
+bem.KoboRange__number = makeBem(bem.KoboRange, 'number', 'span')
+bem.KoboRange__unit = makeBem(bem.KoboRange, 'unit', 'span')
+
+
+bem.KoboRange__progress = makeBem(bem.KoboRange, 'progress', 'div')
+bem.KoboRange__input = makeBem(bem.KoboRange, 'input', 'input')
+
+export enum KoboRangeColours {
+  'default' =  'default',
+  'warning' = 'warning',
+  'teal' = 'teal',
+}
+
+type KoboRangeProps = {
+  max: number,
+  value: number,
+  /** uses time display for all required values */
+  isTime?: boolean,
```

Comment:
When I want `isTime`, what kind of `value` is expected? Seconds, minutes? Would be nice to add it to the comment